pkg/internal/ebpf/common/kafka_detect_transform_test.go

```diff
@@ -23,7 +23,27 @@ func TestProcessKafkaRequest(t *testing.T) {
 			},
 		},
 		{
-			name:  "Produce request",
+			name:  "Fetch request (v12)",
+			input: []byte{0, 0, 0, 52, 0, 1, 0, 12, 0, 0, 1, 3, 0, 12, 99, 111, 110, 115, 117, 109, 101, 114, 45, 49, 45, 49, 0, 255, 255, 255, 255, 0, 0, 1, 244, 0, 0, 0, 1, 3, 32, 0, 0, 0, 30, 37, 158, 231, 0, 0, 0, 156, 1, 1, 1, 0, 53, 99, 48, 57, 45, 52, 52, 48, 48, 45, 98, 54, 101, 101, 45, 56, 54, 102, 97, 102, 101, 102, 57, 52, 102, 101, 98, 0, 2, 9, 109, 121, 45, 116, 111, 112, 105, 99, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 100, 0, 0, 0, 0, 1, 0, 0, 0, 101, 121, 12, 118, 97, 108, 117, 101, 51, 0, 30, 0, 0},
+			expected: &KafkaInfo{
+				ClientID:    "consumer-1-1",
+				Operation:   Fetch,
+				Topic:       "my-topic",
+				TopicOffset: 51,
+			},
+		},
+		{
+			name:  "Fetch request (v15)",
+			input: []byte{0, 0, 0, 68, 0, 1, 0, 15, 0, 0, 38, 94, 0, 32, 99, 111, 110, 115, 117, 109, 101, 114, 45, 102, 114, 97, 117, 100, 100, 101, 116, 101, 99, 116, 105, 111, 110, 115, 101, 114, 118, 105, 99, 101, 45, 49, 0, 0, 0, 1, 244, 0, 0, 0, 1, 3, 32, 0, 0, 0, 33, 62, 224, 94, 0, 0, 30, 44, 1, 1, 1, 0, 1, 70, 99, 111, 110, 115, 117, 109, 101, 114, 45, 102, 114, 97, 117, 100, 100, 101, 116, 101, 99, 116, 105, 111, 110, 115, 101, 114, 118, 105, 99, 101, 45, 49, 45, 50, 51, 48, 98, 51, 55, 101, 100, 45, 98, 101, 57, 102, 45, 52, 97, 53, 99, 45, 97, 52},
+			expected: &KafkaInfo{
+				ClientID:    "consumer-frauddetectionservice-1",
+				Operation:   Fetch,
+				Topic:       "",
```

Comment:
What do you think about making topic equal to `*` when we have an UUID? It seems similar to what we do for HTTP...

Reply:
This will make it clear that we have new protocol when we debug this in production.

Reply:
it will display `process *` as span name, are we ok with that?

Reply:
I think so, * usually in OTel means something high cardinality, so UUID will fit that I think. It will also help us spot the difference when we failed to read, compared to what we knew we couldn't read.